packages/formdescriptioneditors/frontend/sirius-components-formdescriptioneditors/src/Group.tsx

```diff
@@ -466,19 +466,19 @@ export const Group = ({ editingContextId, representationId, formDescriptionEdito
       <div
         data-testid={`Group-DropArea-${group.id}`}
         className={classes.placeholder}
-        onDragEnter={handleDragEnter}
-        onDragOver={handleDragOver}
-        onDragLeave={handleDragLeave}
-        onDrop={handleDrop}
+        onDragEnter={readOnly ? noop : handleDragEnter}
```

Comment:
May be it would have been better/more readable to modify handleDragEnter/handleDragOver/handleDragLeave... instead.
What do you think about it?